docs/people-and-groups/api-keys.md

```diff
@@ -0,0 +1,110 @@
+---
+title: API keys
+---
+
+# API keys
+
+Metabase can create API keys to authenticate programmatic requests to the API.
+
+## Fair warning about Metabase's API
+
+We don't version the [Metabase API](../api-documentation.md). We rarely change API endpoints, and almost never remove them, but if you write code that relies on the API, there's a chance you might have to update your code in the future.
+
+That said, there are times when it's nice to work with the API, like when managing permissions with a large number of people and groups, or bulk archiving, or content creation. So we added the ability to create API keys to authenticate your programmatic requests.
+
+## Create an API key
+
+To create an API key:
+
+1. Click on the **gear** icon in the upper right.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click on the **Authentication** tab on the left menu.
+5. Scroll to **API Keys** and click **Manage**.
+6. Click the **Create API Key** button.
+7. Enter a **Key name**. You can have multiple API keys, so give it a name that will help you remember what you're using the key for.
+8. Select a **Group**. The key will have the same permissions granted to that group.
+9. Click **Create**.
+10. Copy the generated API key and save it somewhere safe. Metabase won't be able to show you the key again. If you lose the key, you'll need to generate a new key.
```

Comment:
very small thing: but maybe we should use you'll need to regenerate the key, because it is the wording we use in-product